packages/dds/tree/README.md

```diff
@@ -9,24 +9,24 @@ A wide variety of possible consumers (across several companies) have overlapping
 which seem like they can best be met by collaborating on a single feature rich tree implementation powered by Fluid.
 The current feature focus is on:
 
-- Semantics:
+-   Semantics:
     -   High quality semantic merges, including "slice" moves of parts of sequences.
```

Comment:
I would expect that most readers would not be familiar with the term "slice move."

Reply:
```suggestion
    -   High quality semantic merges, including moves of parts of sequences (called "slice moves").
```